cloud-optimized-geotiffs/cogs-examples.ipynb

```diff
@@ -9,6 +9,7 @@
    ]
```

Comment:
<p>TODO: for later, we should add a section showing a verification that the values of the data have not changed in the conversion. This question/issue actually came up recently.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Opened issue #99